aws-gitlab-main.yml

```diff
@@ -75,17 +75,60 @@ Parameters:
   GitLabDBMasterUserPasswordParameter:
```

Comment:
oooof bad idea
Look into: https://aws.amazon.com/blogs/mt/using-aws-systems-manager-parameter-store-secure-string-parameters-in-aws-cloudformation-templates/
OR
https://aws.amazon.com/blogs/security/how-to-create-and-retrieve-secrets-managed-in-aws-secrets-manager-using-aws-cloudformation-template/